manager/resolver/maven/src/main/java/org/jboss/forge/furnace/manager/maven/util/MavenRepositories.java

```diff
@@ -26,19 +26,17 @@
  */
 public class MavenRepositories
 {
-   private static final String MAVEN_CENTRAL_REPO = "http://repo1.maven.org/maven2";
+   public static final String MAVEN_CENTRAL_REPO = "http://repo1.maven.org/maven2";
```

Comment:
I think we can remove the public statement here (Make it package-protected) and change the value to be `http://central.maven.org/maven2` ?